### ISSUE TITLE
Messaging tweaks resets bubbles

### ISSUE THREAD

**Issue by nar001** [new-xkit/XKit#1181]
So when I try to chat with people, messaging tweaks resets the bubble colors, which makes the text unreadable, and I have to remove them manually in the settings, and I'm not sure why it's not sticking, thanks!

**Comment by DianeJane** [comment]
Do you mean you remove the presets so that it has tumblr's default colors but then it goes back to messaging tweaks's presets? Because if yes, then this is my issue as well.

**Comment by Wolvan** [comment]
You have to make sure it is
1.) Not an empty field - XKit's storage is programmed to load the default when the fields are empty
2.) Have it be something invalid in CSS so it doesn't actually change the style. You could probably just put in INVALID or something and it should work

**Comment by DianeJane** [comment]
Yeah, setting it to "x" worked for me. Though it seems like a peculiar work around, I would suggest maybe not having the defaults just because it seems more intuitive to leave the field blank. What do you think? I'd submit a pull request if this seems like an acceptable change.

**Comment by Wolvan** [comment]
that would require changing XKit's storage core. Not sure how easy that is gonna be, but go for the PR if you want

**Comment by nightpool** [comment]
@Wolvan removing the default value? That shouldn't be too hard....

**Comment by Wolvan** [comment]
Oh, that's what they meant. Sure, I guess
